src/main/java/com/exceptionless/exceptionlessclient/plugins/preconfigured/RequestInfoPlugin.java

```diff
@@ -73,7 +70,8 @@ private RequestInfo getRequestInfo(HttpRequest request, RequestInfoGetArgs args)
             .httpMethod(request.method())
             .host(request.uri().getHost())
             .path(request.uri().getPath())
-            .port(request.uri().getPort());
+            .port(request.uri().getPort())
+            .headers(request.headers().map());
```

Comment:
Would be good to exclude some of the headers like authorization. See the .NET client.